recipes/cassandra-cpp-driver/all/conanfile.py

```diff
@@ -146,6 +147,7 @@ def package(self):
         copy(self, pattern="LICENSE.txt", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
         cmake = CMake(self)
         cmake.install()
+        fix_apple_shared_install_name(self)
```

Comment:
@uilianries it's overkill, injecting `tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0042"] = "NEW"` is more efficient (indeed it comes from old default CMake policies induced by https://github.com/datastax/cpp-driver/blob/2.17.0/CMakeLists.txt#L1).

Reply:
I've submitted https://github.com/conan-io/conan-center-index/pull/20174

Reply:
Good catch, thank you @SpaceIm